cmds/identityd/main.go

```diff
@@ -355,7 +355,7 @@ func getIdentityMgr(root string, debug bool) (pkg.IdentityManager, error) {
 
 	log.Info().
 		Bool("orphan", env.Orphan).
-		Uint64("farmer_id", uint64(env.FarmerID)).
+		Uint64("farmer_id", uint64(env.FarmID)).
```

Comment:
shouldn't it be `farm_id` now?

Reply:
yeah, the problem is the `cmdline` still uses `farmer_id` which is confusing because it should be farm id. So internally in the code i changed that to FarmID but not changing anything else.